lib/shopify-cli/commands/generate.rb

```diff
@@ -23,8 +23,9 @@ def self.extended_help
           {{bold:Subcommands:}}
 
             {{cyan:page}}: Generate a new page in your app with the specified page name.
-              Usage: {{command:#{ShopifyCli::TOOL_NAME} generate page <pagename>}}
-
+              Usage: {{command:#{ShopifyCli::TOOL_NAME} generate page <pagename>}} or
+                     {{command:#{ShopifyCli::TOOL_NAME} generate page <pagename> --type=TYPE}}
+              Types:
               {{cyan:empty-state}}: generate a new page with an empty state
               {{underline:https://polaris.shopify.com/components/structure/empty-state}}
```

Comment:
Our usual style is to indent everything under a cyan command to aid scannability. Can you indent the URLs?